usr/share/rear/conf/default.conf

```diff
@@ -1943,6 +1943,12 @@ BORGBACKUP_SHOW_RC="no"
 BORGBACKUP_CREATE_SHOW_RC=
 BORGBACKUP_PRUNE_SHOW_RC=
 BORGBACKUP_EXTRACT_SHOW_RC=
+# Pagination for selecting archives:
+# While restoring how many archives should be shown in each step to the user?
+# If more archives are available, cycle through them.
+# Enabled by default with showing up to 10 archives per cycle.
+# To disable it set it 0.
+BORGBACKUP_RESTORE_ARCHIVES_SHOW_NUMBER=10
```

Comment:
SCNR just my frightening nitpicking variable naming questioning ;-)

Could perhaps `BORGBACKUP_RESTORE_ARCHIVES_SHOW_MAX`
be a variable name that better tells what its actual meaning is?

What triggers my interest here is that `NUMBER` is rather meaningless,
imagine tings like `USER_NAME_WORD` or `PATH_TO_FILE_STRING` ;-)

Reply:
@jsmeix thanks for helping with a better name, was already not happy with the `_NUMBER` myself, but your `_MAX` suggestion was not coming to my mind.

Added new commit ad213c3d for it.